src/svgx/SizeIcon.tsx

```diff
@@ -0,0 +1,29 @@
+        fill="#0D0C22"
+      />
+    </Icon>
+  )
+}
+
+export default SizeIcon
```

Comment:
Do we need to add those icons her? They should be available in the ui-kit.

Reply:
For now there no such icons in UI-Kit. I think we can move them in UI-Kit in separate task and update dependencies in block-explorer and here.